version-specific-1.13/ember-internals.js

```diff
@@ -18,7 +18,7 @@ try {
   isStable = emberRequire('ember-htmlbars/keywords/outlet').default.isStable;
 }
 
-export function registerKeywords() {
+export function initialize() {
```

Comment:
I left this as it was in order to more-closely follow liquid-fire where this was borrowed from. I'd prefer to rename it to something that doesn't require monkeying around on the import into the initializer.